scripts/qa_crowbarsetup.sh

```diff
@@ -2883,6 +2883,9 @@ function custom_configuration
             # set discovery root password too
             proposal_set_value provisioner default "['attributes']['provisioner']['discovery']['append']" "\"DISCOVERY_ROOT_PASSWORD=$want_rootpw\""
 
+            # reduce chef intervals
```

Comment:
Shouldn't this say "increase" not "reduce"?  IIUC it's changing it from every 15 mins to every 6 hours.

Reply:
adjusted